packages/instrumentation-kafkajs/src/kafkajs.ts

```diff
@@ -218,7 +218,7 @@ export class KafkaJsInstrumentation extends InstrumentationBase<typeof kafkaJs>
     }
 
     private _endSpansOnPromise<T>(spans: Span[], sendPromise: Promise<T>): Promise<T> {
-        return sendPromise
+        return Promise.resolve(sendPromise)
```

Comment:
Why Promise.resolve? what happens if this promise is still pending? or if it failed?